### PR TITLE
[INTERNAL] Add preliminary guide for v3 migration

### PR DESCRIPTION
**Pull request by matz3**

### CODE REVIEW

File: docs/pages/Builder.md [hunk 1]
```diff
@@ -96,5 +96,12 @@ Since string replacement is a common build step, it can be useful in different c
 
 Available processors are listed [in the API reference](https://sap.github.io/ui5-tooling/api/module-@ui5_builder.processors.html).
 
+## Source Map support
+
+Source Maps will be supported by the upcoming v3.0.0 version which is currently in development.
+Files will be generated automatically for all bundles.
```
Comment (matz3) [inline docs/pages/Builder.md:102]: Maybe we should also link to the relevant PR https://github.com/SAP/ui5-builder/pull/695 ?

File: docs/updates/migrate-v3.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+- debugFileCreator
+- resourceCopier
+- uglifier
+
+As a replacement, the new 'minify' task and 'minifier' processor can be
+used.
+
+Note: The minify task is executed earlier, before the bundling
+process takes place. Existing 'beforeTask' and 'afterTask' configuration of
```
Comment (KlattG) [inline docs/updates/migrate-v3.md:31]:
```suggestion
process takes place. Any existing 'beforeTask' or 'afterTask' configuration of
```

File: docs/updates/migrate-v3.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# Migrate to v3
+
+!!! warning
+    UI5 Tooling v3 is currently in development. Further breaking changes are expected.  
+    The latest development version can be installed via `npm i -D @ui5/cli@next`
+
+## Breaking changes
+
+**All UI5 Tooling Modules: Require Node.js >= 16.13.2 / npm >= 8**
```
Comment (KlattG) [inline docs/updates/migrate-v3.md:9]:
```suggestion
**All UI5 Tooling Modules Require Node.js >= 16.13.2 / npm >= 8**
```

File: docs/updates/migrate-v3.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+| **REMOVED:** ~~uglify~~ |  |  |  |
+| generateVersionInfo | {: .sap-icon-circle-task-2 } |  |  |
+| generateCachebusterInfo | {: .sap-icon-circle-task } |  |  |
+| generateApiIndex | {: .sap-icon-circle-task }^1^ |  |  |
+| generateResourcesJson | {: .sap-icon-circle-task } | {: .sap-icon-circle-task } | {: .sap-icon-circle-task } |
+
+*Enabled by default*
+{: .sap-icon-circle-task-2-before }
+
+*Disabled by default. Can be activated by certain build modes, project configuration or by using the `--include-task` [CLI parameter](../pages/CLI.md#ui5-build). See footnotes where given*
```
Comment (KlattG) [inline docs/updates/migrate-v3.md:65]:
```suggestion
*Disabled by default. Can be activated by certain build modes, the project configuration, or by using the `--include-task` [CLI parameter](../pages/CLI.md#ui5-build). See footnotes where given* 
```

**Review by KlattG — CHANGES_REQUESTED** [review]
Just some minor suggestion, otherwise LGTM

**Comment by KlattG** [comment]
LGTM

**Review by KlattG — APPROVED** [review]

**Review by codeworrior — APPROVED** [review]
LGTM

**Review by codeworrior — APPROVED** [review]
LEBTM ;-)

**Review by ecker — APPROVED** [review]
LGTM, well done!

**Review by KlattG — COMMENTED** [review]
LGTM

**Pull request merged**